manifests/etcd/statefulset.yaml

```diff
@@ -39,7 +39,9 @@ spec:
               --initial-cluster-token etcd-cluster-1 \
               --initial-cluster ${PEERS} \
               --initial-cluster-state new \
-              --data-dir /var/run/etcd/default.etcd
+              --data-dir /var/run/etcd/default.etcd \
+              --auto-compaction-mode=revision \
+              --auto-compaction-retention=1
```

Comment:
1 hour seems extremely low when an ETOS testrun can run for about 24 hours.

Reply:
The mode is set to "revision", so it is one revision, not hours.